main.py

```
@@ -1,6 +1,6 @@
-from PySide6.QtWidgets import QMainWindow, QToolBar, QApplication, QHBoxLayout, QLineEdit, QWidget, QVBoxLayout, QSizePolicy, QScrollArea, QLabel, QTreeView, QStatusBar
-from PySide6.QtCore import Qt, QUrl, QByteArray, Signal
-from PySide6.QtGui import QPixmap
+from PySide6.QtWidgets import QMainWindow, QToolBar, QApplication, QHBoxLayout, QLineEdit, QWidget, QVBoxLayout, QSizePolicy, QScrollArea, QLabel, QTreeView, QStatusBar, QSplitter
+from PySide6.QtCore import Qt, QUrl, QByteArray, Signal, QPropertyAnimation, QEasingCurve
```

Ruff (0.11.9)

`PySide6.QtCore.QUrl` imported but unused

Remove unused import

(F401)